puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/scm/foreman.yaml

```diff
@@ -2,7 +2,8 @@
     name: foreman
     scm:
       - git:
-         url: git://github.com/theforeman/foreman.git
-         wipe-workspace: true
-         branches:
-          - 'develop'
+          url: git://github.com/theforeman/foreman.git
+          wipe-workspace: true
+          basedir: foreman
```

Comment:
Changing this will mean you also need to update the paths in publishers in test_develop.